### ISSUE TITLE
Screenreader Support

### ISSUE THREAD

**Issue by jbrooksuk** [jgthms/bulma#150]
Bootstrap has `sr-only` which will show an element for screenreaders only. Bulma seems to be missing this.

**Comment by TroyAlford** [comment]
Love this suggestion. I think Bootstrap is likely to be one of the primary points of comparison for people who are considering using Bulma. While full feature-parity is probably not necessary, some of the nice semantics around screen readers and accessibility might be important.

**Comment by jbrooksuk** [comment]
@TroyAlford I implemented this https://github.com/jgthms/bulma/pull/153, what do you think?

**Comment by TroyAlford** [comment]
@jbrooksuk love it, and the fact that it mirrors `.sr-only` and `.sr-only-focusable` [from bootstrap](https://github.com/twbs/bootstrap/blob/e38f066d8c203c3e032da0ff23cd2d6098ee2dd6/less/scaffolding.less#L125-L150).

**Comment by jbrooksuk** [comment]
:+1:

**Comment by ghengeveld** [comment]
Any update on this?

**Comment by viki53** [comment]
Same here, this should be a standard feature in Bulma. Accessibility is not an option, it should be available from the start.

Why not merge #153 if it works?

**Comment by Vinnl** [comment]
@jgthms You're currently assigned to this - are you still working on it? If not, are there pointers for what a solution should look like/adhere to, so other people can contribute?

**Comment by taciturasa** [comment]
I will not be using Bulma unless it is built with accessibility in mind. A non-accessible web is inexcusable and worthless.

**Comment by EmmanuelSimond** [comment]
@viki53 "this should be a standard feature in Bulma. Accessibility is not an option, it should be available from the start."

Bulma is v. 0.7.1+ only. Sure Accessibility will be here in v.1.0

**Comment by EmmanuelSimond** [comment]
@taciturasa "(...) built with accessibility in mind"

Bulma is v. 0.7.1+ only. Sure Accessibility will be here in v.1.0

**Comment by anton-g** [comment]
This have been added in #1764. The documentation on bulma.io is already updated (why?) but it won't be available until @jgthms releases 0.7.2 on npm.

**Comment by Muhnad** [comment]
@jbrooksuk thanks for reporting this issue. the issue from 2016 and we added it to bulma lately I'm so sorry for that and for everyone who use/ed bulma.

I think we need to report all screenreader issues in this Issue to can figure out how to start fixing it and add it to bulma.
